dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGVisualizeLauncher.java

```diff
@@ -116,6 +127,25 @@ protected void generateDOTofCFGWithoutAnalysis(
         generateDOTofCFG(inputFile, outputDir, method, clas, pdf, verbose, null);
     }
 
+    /**
+     * Generate the String representation of the CFG for a method without analysis.
```

Comment:
What does "without analysis" mean?  It's here and in the method name.

I realize this phrase exists elsewhere in the code (it's not your invention), but I don't understand it in the other places and its documentation could be improved throughout.  Thanks!

Reply:
"without analysis" means that `analysis=null`. So `analysis.performAnalysis(cfg);` will not be executed. Only the control flow graph will be visualized.

Before this PR, you can generate the dot representation of the control flow graph by:

```
java -Xbootclasspath/p:$CHECKERFRAMEWORK/checker/dist/javac.jar \
  -cp $CHECKERFRAMEWORK/checker/dist/checker.jar \
  org.checkerframework.dataflow.cfg.CFGVisualizeLauncher \
Test.java cfg -class Test -method test
```

Then the dot files will be generated. (You can also add `-pdf` to generate pdf files.)

But there is no way to see the string representation from the command line.

After this PR (add -string at the end of the command):

```
java -Xbootclasspath/p:$CHECKERFRAMEWORK/checker/dist/javac.jar \                                                                           checker-framework -> typetools-improve-cfgvisualizelauncher $ ?
  -cp $CHECKERFRAMEWORK/checker/dist/checker.jar \
  org.checkerframework.dataflow.cfg.CFGVisualizeLauncher \
Test.java cfg -class Test -method test -string
```

A string graph will be printed:

```
2 -> 3
3 -> 4
4 -> 8
4 -> 10
8 -> 0
10 -> 0

2:
<entry>

3:
a   [ VariableDeclaration ]
1   [ IntegerLiteral ]
a = 1   [ Assignment ]
b   [ VariableDeclaration ]
2   [ IntegerLiteral ]
b = 2   [ Assignment ]
c   [ VariableDeclaration ]
3   [ IntegerLiteral ]
c = 3   [ Assignment ]
a   [ LocalVariable ]
0   [ IntegerLiteral ]
(a > 0)   [ GreaterThan ]

4:
ConditionalBlock: then: 8, else: 10

8:
d   [ VariableDeclaration ]
a   [ LocalVariable ]
c   [ LocalVariable ]
(a + c)   [ NumericalAddition ]
d = (a + c)   [ Assignment ]

10:
e   [ VariableDeclaration ]
a   [ LocalVariable ]
b   [ LocalVariable ]
(a + b)   [ NumericalAddition ]
e = (a + b)   [ Assignment ]

0:
<exit>
```

I think this may be convenient when we only want to visualize a simple program.

Reply:
Thanks for the explanation.  I wrote some text for the manual (which is what I really wanted.)  Please check to see that it is accurate.

I am not convinced I would want to use this, but I don't object to providing the feature.

Could you please document it in the manual as well as in the changelog?